client/app/types/env.d.ts

```diff
@@ -1,5 +1,9 @@
-declare module '@env' {
-
-export const EXPO_IP: string;
-export const API_KEY, AUTH_DOMAIN, PROJECT_ID, STORAGE_BUCKET, MESSAGING_SENDER_ID, APP_ID: string;
+declare module "@env" {
```

Comment:
Take a look at the main repository's config md. You will see that we updated it to throw away a bunch of these environment variables. Please only import updated required ones.